
const placeanorder = async (req,res) => {
    try {
        
    } catch (error) {
        
    }   
    
}